org.code.toboggan.network/src/org/code/toboggan/network/notification/clientcorelisteners/file/FileMoveNotificationHandler.java

```diff
@@ -27,13 +28,16 @@
 	
 	@Override
 	public void handleNotification(Notification notification) {
-		FileMoveNotification n = (FileMoveNotification) notification.getData();
+		FileMoveNotification n = (FileMoveNotification) notification.getData(); 
 		logger.info("Received file move notification for " + notification.getResourceID());
 		Set<ICoreExtension> extensions = extMgr.getExtensions(NetworkExtensionIDs.FILE_MOVE_NOTIFICATION_ID, IFileMoveNotificationExtension.class);
+		 
+		File file = storage.getFile(notification.getResourceID());
+		Path newAbsolutePath = CoreActivator.getSessionStorage().getProjectLocation(file.getProjectID()).resolve(n.newPath).resolve(file.getFilename());
```

Comment:
The toAbsolutePathFromRelative isn't, but the toStringRelativePath is. Also, I generally don't delete util methods unless there's a clear reason to do so.

It was acting incorrectly in this case, since the newPath had no concept of which project directory it was attached to. Thus it was treating the project-relative filepaths as a workspace relative filepath.